benchmarks/WebJobs.Script.Benchmarks/SanitizerBenchmarks.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.Azure.WebJobs.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Benchmarks
+{
+    public class SanitizerBenchmarks
+    {
+        [Benchmark]
+        public void Sanitize()
+        {
+            Sanitizer.Sanitize("testprotocol://name:password@address:1111");
+        }
+    }
+}
```

Comment:
Do we need to add this?

Reply:
@brettsam recommended adding this. I am not familiar with it but it seems that benchmarks give us the speed and memory footprint (allocations) of the specific method, so we can come in and do tweaks and compare.

Reply:
It can provide that info. But that is useful when we are comparing 2 different implementations. So I am not sure what value this will add by itself.

Reply:
By 2 different implementations, do you mean to also call `Sanitizer.Sanitze("<input>")` with some different input? And then we will get comparison between 2 inputs?